README.md

```diff
@@ -249,21 +256,22 @@ let is_sql = client.query(query_is_sql.unwrap()).unwrap();
 // {[":mysql", "MySQL"], [":postgres", "Postgres"]} BTreeSet
 ```
 
-[`Action`](https://docs.rs/transistor/1.0.0-beta.3/transistor/http/enum.Action.html) is an enum with a set of options to use in association with the function `tx_log`:
+[`Action`](https://docs.rs/transistor/1.0.0/transistor/http/enum.Action.html) is an enum with a set of options to use in association with the function `tx_log`:
 * [`Put`](https://opencrux.com/docs#transactions-put) - Write a version of a document
 * [`Delete`](https://opencrux.com/docs#transactions-delete) - Deletes the specific document at a given valid time
 * [`Evict`](https://opencrux.com/docs#transactions-evict) - Evicts a document entirely, including all historical versions (receives only the ID to evict)
 * [`Match`](https://opencrux.com/docs#transactions-match) - Matches the current state of an entity, if the state doesn't match the provided document, the transaction will not continue
 
-[`Query`](https://docs.rs/transistor/1.0.0-beta.3/transistor/types/query/struct.Query.html) is a struct responsible for creating the fields and serializing them into the correct `query` format. It has a function for each field and a `build` function to help check if it is correctyly formatted.
+[`Query`](https://docs.rs/transistor/1.0.0/transistor/types/query/struct.Query.html) is a struct responsible for creating the fields and serializing them into the correct `query` format. It has a function for each field and a `build` function to help check if it is correctyly formatted.
 * `find` is a static builder function to define the elements inside the `:find` clause.
 * `where_clause` is a builder function that defines the vector os elements inside the `:where []` array.
 * `order_by` is a builder function to define the elements inside the `:order-by` clause.
 * `args` is a builder function to define the elements inside the `:args` clause.
 * `limit` is a builder function to define the elements inside the `:limit` clause.
 * `offset` is a builder function to define the elements inside the `:offset` clause.
+* `with_full_results` is a builder function to define the flag `full-results?` as true. This allows your `query` response to return the whole document instead of only the searched keys. The result of the Query `{:query {:find [?user ?a] :where [[?user :first-name ?a]] :full-results? true}}` will be a `BTreeSet<Vec<String>>` like `([{:crux.db/id :fafilda, :first-name "Jorge", :last-name "Klaus"} "Jorge"])`, so the document will need further EDN parsing to become the document's struct.
```

Comment:
What do you think of a name with something more related to `document` or `entity`?
Like `full_entity_results` or `full_document`, idk just wondering.

Reply:
My opinion, as we are not passing arguments to this functions, the name declaration states that `:full-results? true`, if we were to change to `full_results` I would expect it to receive a boolean as argument, which seems like an over-engineering for this field

Reply:
Oh, I was thinking of just `full_results` without passing a boolean, because it is redundant as you've said. But we can use the `with` prefix if you prefer it.

Reply:
same logic goes `with_authorization`, but there we have to passa auth code 🤔

Reply:
Hmmm, now you made me like my argument more hahahahah
Let's look at two examples, so we can decide this visually:

```rust
let q = Query::find(vec!["?p1"])
    .unwrap()
    .where_clause(vec!["?p1 :first-name n", "?p1 :last-name \"Jorge\""])
    .unwrap()
    .with_full_results()
    .build();
```

vs

```rust
let q = Query::find(vec!["?p1"])
    .unwrap()
    .where_clause(vec!["?p1 :first-name n", "?p1 :last-name \"Jorge\""])
    .unwrap()
    .full_results()
    .build();
```

Yeah, I prefer just `full_results`, what do you think?
The `with_` prefix sounds like it will receive a value.

Reply:
I think the opposite haushasuhas, that `full_results` should receive an argument 😂

Reply:
just `full_results` sounds empty and incomplete 🤔